src/modules/auth/screens/login-screen.tsx

```diff
@@ -1,7 +1,32 @@
-import { Text, View } from "@/ui/core"
+import { Image, Text, View, ImageBackground, SafeAreaView, Button, TextInput } from '../../../ui/core'
+import images from '../../../ui/assets/images';
+import type { AuthStackScreenProps } from '../../../navigation/types';
+import { TouchableOpacity } from 'react-native'
 
-export const LoginScreen = () => {
-    return (<View>
-        <Text>Login Screen</Text>
-    </View>)
+type ScreenProps = {} & AuthStackScreenProps<'Login'>;
```

Comment:
Is the `{}` here in case you add some other prop later?

Reply:
Done